content/docs/next-release/atm.md

```diff
@@ -0,0 +1,304 @@
+  - For example, if set to `3600000` (1 hour), the query would span from `endTs - 1 hour` to `endTs`.
+  - Optional with default: 3600000 (1 hour).
+
+step = 'step=' intValue
+  - The duration, in milliseconds, between data points of the query results.
+  - For example, if set to 5s, the results would produce a data point every 5 seconds from the `endTs - lookback` to `endTs`.
+  - Optional with default: 5000 (5 seconds).
+
+ratePer = 'ratePer=' intValue
+  - The duration, in milliseconds, in which the per-second rate of change is calculated for a cumulative counter metric.
+  - Optional with default: 600000 (10 minutes).
+
+spanKinds = spanKind | spanKind '&' spanKinds
+spanKind = 'spanKind=' spanKindType
+spanKindType = 'unspecified' | 'internal' | 'server' | 'client' | 'producer' | 'consumer'
+  - The list of spanKinds to include in the metrics selection filter, which are logically 'OR'ed.
+  - Optional with default: 'server'
+```
+
+##### Responses
+
+The response data model is based on [OpenMetrics' `MetricFamily`][openmetrics.proto]
+
+For example:
+```
+{
+  "name": "service_call_rate",
+  "type": "GAUGE",
+  "help": "calls/sec, grouped by service",
+  "metrics": [
+    {
+      "labels": [
+        {
+          "name": "service_name",
+          "value": "driver"
+        }
+      ],
+      "metricPoints": [
+        {
+          "gaugeValue": {
+            "doubleValue": 0.005846808321083344
+          },
+          "timestamp": "2021-06-03T09:12:06Z"
+        },
+        {
+          "gaugeValue": {
+            "doubleValue": 0.006960443672323934
+          },
+          "timestamp": "2021-06-03T09:12:11Z"
+        },
+...
+  ```
+
+If the `groupByOperation=true` parameter is set, the response will include the operation name in the labels like so:
+```
+      "labels": [
+        {
+          "name": "operation",
+          "value": "/FindNearest"
+        },
+        {
+          "name": "service_name",
+          "value": "driver"
+        }
+      ],
+```
+
+#### Min Step Query
+
+`/api/metrics/minstep`
+
+Gets the min time resolution supported by the backing metrics store, in milliseconds, that can be used in the `step` parameter.
+e.g. a min step of 1 means the backend can only return data points that are at least 1ms apart, not closer.
+
+## Troubleshooting
+
+### Service/Operation missing in Monitor Tab
+
+If the service/operation is missing in the Monitor Tab, but visible in the Jaeger
+Trace search service and operation drop-downs menus, a common cause of this is
+the default `server` span kind used in metrics queries.
+
+The service/operations you are not seeing could be from spans that are non-server
+span kinds such as client or worse, `unspecified`. Hence, this is an instrumentation
+data quality issue, and the instrumentation should set the span kind.
+
+The reason for defaulting to `server` span kinds is to avoid double-counting
+both ingress and egress spans in the `server` and `client` span kinds, respectively.
```

Comment:
are there some internal metrics worth monitoring that might indicate infra issues, like not being able to reach Prom server?